kuzzleio/sdk-cpp
# Fix wrapping of options and room options

## What does this PR do ?

This PR add constructors for `options`, `query_options` and `room_options` structs to set default values.

https://github.com/kuzzleio/sdk-c/pull/27 :arrow_left: :large_blue_circle:  :arrow_right: https://github.com/kuzzleio/sdk-java/pull/22

### How should this be manually tested?

  - Step 2 : Make the sdk `docker run --rm -it --network ci_default --link kuzzle -v "$(pwd)":/mnt kuzzleio/sdk-cross:gcc make clean all`
  - Step 3 :  Get this snippet: https://gist.github.com/Aschen/84b33bf47e883275c690cdaf74c39f98
  - Step 4 :  Compile and run `g++ -std=c++11 source.cpp -Isdk-c/include -Isdk-c/build/ -Iinclude -L./build/ -lkuzzlesdk -lpthread && LD_LIBRARY_PATH=./build ./a.out`
  - Step 5 : You should see `120000`, `'all'` and `10` printed to stdout
